draft-ietf-ice-rfc5245bis.xml

```diff
@@ -3023,49 +2772,31 @@ all media streams using those candidates.
 
 <section anchor="sec-restart" title="ICE Restarts">
 <t>
-An agent MAY restart ICE processing for an existing media stream.
-An ICE restart will cause all previous states, excluding the roles
-of the agents, of ICE processing to be flushed and checks to start anew.
-The only difference between an ICE restart and a brand new media session
-is that during the restart media can continue to be sent to the
-previously validated pair, and that a new media session always requires
-the roles to be determined.
+An agent MAY restart ICE for existing media streams.  An ICE restart
+causes all previous state of the media streams, excluding the roles of
+the agents to be flushed.  The only difference between an ICE restart
+and a brand new media session is that during the restart media can
+continue to be sent, and that a new media session always requires the
+roles to be determined.
 </t>
```

Comment:
Yeah, that's redundant, though.  The spec is clear that you can never send except on a validated pair.  

Plus it says "THE previously validated pair", when there might be many.  And candidate pair selection/nomination might not have concluded on the ICE session that got restarted.